edx_proctoring/templates/proctoring/proctoring_launch_callback.html

```diff
@@ -1,11 +1,205 @@
+        padding: 20px;
+      }
+      hr {
+        border:none;
+        border-bottom: 1px solid rgb(217, 225, 228);
+      }
+      .footer{
+        border-top: 1px solid rgb(215, 224, 227);
+        background: rgb(229, 235, 237);
+        padding: 20px 50px;
+        margin: 0 -50px;
+      }
+      .footer button{
+        background: rgb(16, 118, 189);
+        color: white;
+        font-size: 17px;
+        padding: 7px 60px;
+        border: none;
+      }
+      .footer button:hover {
+        cursor: pointer;
+        background: rgb(8, 92, 150);
+      }
+    </style>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  {% load i18n %}
+  <body>
+    <div class="steps-bk">
+      <div id="steps-wrapper">
+        <ul class="steps">
+          <li class="photoid completed">Take ID Photo </li>
+          <li class="roomscan completed">Record Room Scan</li>
+          <li class="photo completed">Take User Photo</li>
+          <li class="end active" >Enter Exam</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="steps-container">
+      <div class="body-container">
+        <h3>{% blocktrans %} Your Proctoring Set Up is Complete {% endblocktrans %}</h3>
+        <p>{% blocktrans %} You may proceed and enter exam now {% endblocktrans %}</p>
+
+        <div class="alert">
+          {% blocktrans %} Please leave the window open in the background {% endblocktrans %}
+        </div>
+        <h5> {% blocktrans %} Remember to abide by the guidelines and rules of your proctored exam, including: {% endblocktrans %}</h5>
+        <p>
+          {% blocktrans %}
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+            Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+            Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim.
+          {% endblocktrans %}
+        </p>
+        <hr>
+        <p>
+          {% blocktrans %}
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+            Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+            Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim.
+          {% endblocktrans %}
+        </p>
+        <hr>
+        <p>
+          {% blocktrans %}
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
+            Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+            Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim.
+          {% endblocktrans %}
+        </p>
+      </div>
+    <div class="footer">
+      <button>
+        {% blocktrans %} Enter My Exam {% endblocktrans %}
+      </button>
+    </div>
```

Comment:
Thanks, @chrisndodge. With no button, we would be starting the timer in edx platform at the same time as SS's own "timer"? I can't remember if there is a separate "Start exam" button on the edX end. If not, it seems like a less than ideal student experience to have to race back to the edX page to avoid losing time. And it probably means we wouldn't want to put too much text on this page that would take time to read. Or is there a buffer amount of time to allow the student to switch windows and get set?